packaging/os/apt_repository.py

```diff
@@ -81,6 +87,12 @@
 # On Ubuntu target: add nginx stable repository from PPA and install its signing key.
 # On Debian target: adding PPA is not available, so it will fail immediately.
 apt_repository: repo='ppa:nginx/stable'
+
+# Call the apt-get update to get all recent package lists with their dependencies.
+apt_repository: repo='deb http://archive.canonical.com/ubuntu hardy partner' update_cache=yes
+
+# Call the apt-get update to get the recent package lists only for the added repository.
+apt_repository: repo='deb http://archive.canonical.com/ubuntu hardy partner' update_only_added=yes
```

Comment:
It seems to me that this example will fail with "You must set update_cache=yes to use update_only_added option" if used without modification.